### PR TITLE
Adjusting mortality and adding susceptibility

### PR DESCRIPTION
**Pull request by paulbessell**
Tweaking some disease behaviours in line wiht recent LSHTM publication

### CODE REVIEW

File: src/main/java/uk/co/ramp/covid/simulation/population/Person.java [hunk 1]
```diff
@@ -61,6 +65,12 @@ public Person(int age, Sex sex) {
         this.quarantineProb = PopulationParameters.get().personProperties.pQuarantinesIfSymptomatic;
         this.goesToHospitalInPhase2 = CovidParameters.get().hospitalisationParameters.pPhase2GoesToHosptial.sample();
         this.personId = nPeople++;
+        
+        this.covidMortalityAgeAdjustment = Math.pow((double) age / 85.0, 2.0);
+        
+        if(age <= 20) this.covidSusceptibleVal = PopulationParameters.get().personProperties.pSusceptibleChild;
```
Comment (bobtaylor12) [inline src/main/java/uk/co/ramp/covid/simulation/population/Person.java:71]: Should this be <18

File: src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java [hunk 2]
```diff
@@ -14,7 +14,7 @@
     public CareHome(Size s) {
         super(s);
 
-        transConstant = PopulationParameters.get().buildingProperties.careHomeTransmissionConstant;
+        transAdjustment = PopulationParameters.get().buildingProperties.careHomeTransmissionConstant;
```
Comment (bobtaylor12) [inline src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java:17]: transAdjustment seems mis-named to me, or is it careHomeTransmissionConstant that is misnamed?

File: src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java [hunk 3]
```diff
@@ -75,12 +75,13 @@ public boolean isDead() {
     }
     
     private void setSymptomatic() {
-        if(ccase.getAge() >= 60) {
+        if(ccase.getAge() >= 70) {
             // This is set to 60 because the probability is from the Diamond Princess where people were aged > 60
             symptomaticCase = CovidParameters.get().diseaseParameters.pSymptomaticCasePensioner.sample();
-        } else {
-            symptomaticCase = CovidParameters.get().diseaseParameters.pSymptomaticCase.sample();
+        } else if(ccase.getAge() < 70 && ccase.getAge() > 20) {
```
[the remark this answers is not in the document]
Comment (paulbessell) [reply src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java:81]: No. The paper gave the value from age 20. I will annotate with a comment

File: src/main/java/uk/co/ramp/covid/simulation/population/Person.java (hunk 1 above)
Comment (paulbessell) [reply src/main/java/uk/co/ramp/covid/simulation/population/Person.java:71]: Same as above, the paper gave parameters for people up to 20

File: src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java (hunk 2 above)
Comment (paulbessell) [reply src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java:17]: Discussed this wiht Blair. Adjusted to transAdjustment to make it consistent wiht other Place() Classes. Will add an issue to sort htes eout

File: src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (bobtaylor12) [reply src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java:81]: Ok. Should we change our age definitions of Child and Adult to be consistent?

[the remark this answers is not in the document]
Comment (BlairArchibald) [reply src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java:81]: We might want to make these parameters just so we have an external reference (for things like data pipeline) that this behavior is in here. Even if we don't change them in practice.

[the remark this answers is not in the document]
Comment (BlairArchibald) [reply src/main/java/uk/co/ramp/covid/simulation/covid/Covid.java:81]: We could also consider the "Pensioner" etc controlling this, and Covid asking ccase what the parameter should be for that specific person (seems to be some of this below already, e.g. Mortality age adjust?)

File: src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java (hunk 2 above)
Comment (BlairArchibald) [reply src/main/java/uk/co/ramp/covid/simulation/place/CareHome.java:17]: Agree we should keep this consistent now and fix the names in a future (relatively high priority) issue

**Pull request merged**